<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="2" total="4" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-08-22T15:11:31 PDT" name="Suite" finished-at="2021-08-22T15:12:00 PDT" duration-ms="29468">
    <groups>
    </groups>
    <test started-at="2021-08-22T15:11:31 PDT" name="Test" finished-at="2021-08-22T15:12:00 PDT" duration-ms="29468">
      <class name="com.hrms.testCases.HWTest">
        <test-method is-config="true" signature="generateReport()[pri:0, instance:com.hrms.testCases.HWTest@2f7298b]" started-at="2021-08-22T15:11:31 PDT" name="generateReport" finished-at="2021-08-22T15:11:31 PDT" duration-ms="232" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateReport -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.hrms.testCases.HWTest@2f7298b]" started-at="2021-08-22T15:11:31 PDT" name="setUp" finished-at="2021-08-22T15:11:35 PDT" duration-ms="3940" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="loginCredentials()[pri:0, instance:com.hrms.testCases.HWTest@2f7298b]" started-at="2021-08-22T15:11:35 PDT" name="loginCredentials" finished-at="2021-08-22T15:11:52 PDT" duration-ms="17437" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginCredentials -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.hrms.testCases.HWTest@2f7298b]" started-at="2021-08-22T15:11:52 PDT" name="tearDown" finished-at="2021-08-22T15:11:53 PDT" duration-ms="762" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="writeReport()[pri:0, instance:com.hrms.testCases.HWTest@2f7298b]" started-at="2021-08-22T15:11:59 PDT" name="writeReport" finished-at="2021-08-22T15:12:00 PDT" duration-ms="1691" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- writeReport -->
      </class> <!-- com.hrms.testCases.HWTest -->
      <class name="com.hrms.testCases.AddEmployeeHW">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.hrms.testCases.AddEmployeeHW@188715b5]" started-at="2021-08-22T15:11:53 PDT" name="setUp" finished-at="2021-08-22T15:11:56 PDT" duration-ms="2674" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="login()[pri:1, instance:com.hrms.testCases.AddEmployeeHW@188715b5]" started-at="2021-08-22T15:11:56 PDT" name="login" finished-at="2021-08-22T15:11:58 PDT" duration-ms="1944" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[login failed expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: login failed expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at com.hrms.testCases.AddEmployeeHW.login(AddEmployeeHW.java:19)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.hrms.testCases.AddEmployeeHW@188715b5]" started-at="2021-08-22T15:11:58 PDT" name="tearDown" finished-at="2021-08-22T15:11:59 PDT" duration-ms="719" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.hrms.testCases.AddEmployeeHW -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
